web/webkit/src/test/scala/net/liftweb/http/rest/RestHelperSpec.scala

```diff
@@ -17,20 +20,46 @@ class RestHelperSpec extends WebSpec(RestHelperSpecBoot.boot _) {
   sequential  // This is important for using SessionVars, etc.
 
   "RestHelper" should {
-    val testUrl = "http://foo.com/api/info"
+    val testOptionsUrl = "http://foo.com/api/info"
+    val testFutureUrl = "http://foo.com/api/futured"
 
-    val testOptionsReq = new MockHttpServletRequest(testUrl){
+    val testOptionsReq = new MockHttpServletRequest(testOptionsUrl){
       method = "OPTIONS"
     }
 
+    val testFutureReq = new MockHttpServletRequest(testFutureUrl){
+      method = "GET"
+    }
+
     "set OPTIONS method" withReqFor testOptionsReq in { req =>
       req.options_? must_== true
     }
 
     "give the correct response" withReqFor testOptionsReq in { req =>
-      RestHelperSpecRest(req)() match {
-        case Full(OkResponse()) => success
-        case other =>              failure("Invalid response : " + other)
+      RestHelperSpecRest(req)() must beLike {
+        case Full(OkResponse()) => ok
+      }
+    }
+
+    "respond async with LAFutures" withReqFor testFutureReq in { req =>
```

Comment:
I need to change this to say “respond async with something that CanResolveAsync”. No need to test whether there are `CanRespondAsync`s for both `LAFuture` and `Future`, since that's verified elsewhere.